README.md

```diff
@@ -140,6 +161,40 @@ Modules are also supported:
 cdk-import AWSQS::CheckPoint::CloudGuardQS::MODULE
 ```
 
+## AWS Service Catalog Usage
+
+The cdk-import tool generates a user friendly version of a provisioned product
+that can be used like a normal cdk construct within a cdk app.
+You can currently either specify a specific product version or generate all available products
+The tool will call APIs and attempt to resolve default artifact and launch path for a product,
```

Comment:
Since SC permissions work differently is it worth calling out what kind of access you need?

Reply:
sure that is a good idea

Reply:
added a line about it